src/client/Dashboard.tsx

```diff
@@ -895,6 +1012,60 @@ class Dashboard<Props> extends React.Component<Props, IDashboardState> {
     return str;
   };
 
+  private getPRRequestParams = (deployment: IDeployment) => {
+    const query: any = {};
+    if (!deployment.hldRepo) {
+      return "";
+    }
+    const repo: IAzureDevOpsRepo | IGitHub | undefined = getRepositoryFromURL(
+      deployment.hldRepo
+    );
+    if (repo && "username" in repo) {
+      query.username = repo.username;
+      query.reponame = repo.reponame;
+      query.pr = deployment.pr;
+    } else if (repo && "org" in repo) {
+      query.org = repo.org;
+      query.project = repo.project;
+      query.repo = repo.repo;
+      query.pr = deployment.pr;
+    }
+    let str = "";
+    // tslint:disable-next-line: forin
+    for (const key in query) {
+      if (str !== "") {
+        str += "&";
+      }
+      str += key + "=" + encodeURIComponent(query[key]);
+    }
+    return str;
+  };
+
+  private getPRs = () => {
+    try {
+      const state = this.state;
+      this.state.deployments.forEach(deployment => {
+        if (deployment.pr) {
+          const queryParams = this.getPRRequestParams(deployment);
+          if (queryParams !== "") {
+            HttpHelper.httpGet("/api/pr?" + queryParams).then((data: any) => {
+              const pr = data.data as IPullRequest;
```

Comment:
please create a new interface and avoid using `any` like
```
interface IQueryResult {
  data: IPullRequest;
}
```

Reply:
Removed the use of any at the moment, but the entire repository is using anys in many different places. Will have to create a task to clean those up.

Reply:
@samiyaakhtar let's identify all the places and write it up in a task. The longer we avoid the harder it is to fix